test/e2e/util.go

```diff
@@ -565,17 +565,21 @@ func getRouteHost(routeName, namespace string) (string, error) {
 	return out[1 : len(out)-1], nil
 }
 
-func newOAuthProxyService() *corev1.Service {
+func newOAuthProxyService(suffix string) *corev1.Service {
+	name := "proxy"
+	if suffix != "" {
```

Comment:
nit to keep conventions:
```suggestion
	if len(suffix) > 0 {
```